employee.php

```diff
@@ -0,0 +1,91 @@
+<html>
+<form method="POST" action="employee.php">
+        Customer Name: <input type="text" name="custName"> <br>
+        Customer Address: <input type="text" name="custAddr"> <br>
+        <input type="submit" value="Find Customer" name="findCust"></p>
+        <br>
+        <br>
+	Location:
+	<select name="loc">
+		  		<?php 
+		  			require_once 'util.php';
+		  			$util2 = new Util;
+					$debug = True;
+					if ($debug) {
+					}
+		  			$db_conn = OCILogon("ora_j7l8", "a31501125", "ug");
```

Comment:
this is logging into a different db then you are using below https://github.com/banafshehderayat/304project/pull/4/files#diff-e20c71aea6d57e551c2be01ec4738657R45